### DISCUSSION TITLE
Integer-based Indexing

### DISCUSSION THREAD

**Discussion by ilan-gold** [LDeakin/zarrs#49] [Q&A]
Is there any plan for an integer-based chunk (or array) indexer i.e., one that doesn't rely entirely on ranges (+ chunk indices if chunks)?  It seems like `ndarray` is already a dep so this shouldn't be too hard (knowing nothing about ndarray at all)?  I'm going to prototype something any so if you're interested, could be a contribution, and we can make this an issue.

But if it already exists, please point me to it!

**Comment by LDeakin** [comment]
It doesn't exist, but sounds like a handy addition!

**Comment by ilan-gold** [comment]
Moving to issue!
